app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -149,7 +149,7 @@ def shutdown_middleware_server(ems_ref, _params)
     end
 
     def suspend_middleware_server(ems_ref, params)
-      timeout = params.fetch ':timeout' || 0
+      timeout = params[':timeout'] || 0
```

Comment:
another possibility would be:
`timeout = params.fetch(''somekey",0) `
where 0 stands for default value to return if key "somekey" doesn't exist